packages/sequelize-data-provider/src/sequelizeDataProvider.js

```diff
@@ -232,6 +245,22 @@ async function getEntitiesConnection(entityName, args) {
   };
 }
 
+async function handleEntityRelationsPreCreate(entityName, entityToCreate, isCreateMany = false) {
+  const listRelations = [];
+  await async.each(Object.keys(entityToCreate), async entityField => {
+    if (isObject(entityToCreate[entityField])) {
+      if (isCreateMany && entityToCreate[entityField].create) {
+        // In order to enable this need to verify that nested mutations work correctly with the bulk create scenario
```

Comment:
I don't understand enough the implications of running nested create inside bulk create. For now throwing an unsupported error and we can discuss further.